src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(health, name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  removeDead() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
```

Comment:
[METHOD PLACEMENT] - The `removeDead` method should not be an instance method of the `Animal` class. It should be a static method since it operates on the static `Animal.alive` array. Please change `removeDead` to be a static method.